code/part-two/processor/test/00-MojiHandler.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const expect = require('chai').expect;
+const { InvalidTransaction } = require('sawtooth-sdk/processor/exceptions');
+
+const MojiHandler = require('../handler').MojiHandler;
```

Comment:
this might be an opportunity to apply destructuring more consistently. (line 3 as well.) also applies to other files

Reply:
Hrmm, yes, that is very inconsistent. I was trying to avoid destructuring to cut down on ES6 syntax for the students. I finally gave in on InvalidTransaction, because it ends up being over 80 characters without it. I'll let you make the call. Everything destructured, or everything dot notation?

Reply:
@delventhalz I guess we might as well go everything destructured; maybe we can include a link to http://es6-features.org/ (or similar) in the `README` later if we feel like it's necessary